index.bs

```diff
@@ -109,11 +111,29 @@ and taking the value of the cumulative layout shift score at that time.
 
 This strategy is illustrated in the usage example.
 
+Source attribution {#source-attribution}
+----------------------------------------
+
+In addition to the layout shift value, the API reports a sampling of up to five
```

Comment:
I was asked to remove the non-normative note text on the subsections because it was already applied to the entire "1. Introduction" section.  Do you think each subsection should separately state it?

"We" in this case is me, the spec author :)  How do you think it should be rephrased?